crates/revm/src/lib.rs

```diff
@@ -6,6 +6,9 @@ mod evm_impl;
 mod inspector;
 mod journaled_state;
 
+#[cfg(feature = "with-serde")]
```

Comment:
```suggestion
#[cfg(all(feature = "with-serde", not("serde" )]
```

Added `all (not serde), (with serde)` as tests turn all features on and this compile error would trigger.

Reply:
ahh mb! overlooked this